### PR TITLE
hub: Fix Prisma in deployed environments

### PR DESCRIPTION
**Pull request by backspace**
This updates the Webpack and Docker builds to include the files needed to use Prisma.

With this deployed, Prisma works as expected:

```
curl \
 -s -S -H "Accept: application/vnd.api+json" \
 -H "Accept-Encoding: gzip" \
 -H "Content-Type: application/vnd.api+json" \
 "https://hub-staging.stack.cards/api/prepaid-card-patterns"
{"data":[{"id":"3b8b436b-a99e-4346-ab20-3ba5f9963321","type":"prepaid-card-patterns","attributes":{"pattern- …
```

### CODE REVIEW

**Comment by backspace** [comment]
Maybe we need something like [this](https://gist.github.com/malteneuss/a7fafae22ea81e778654f72c16fe58d3#file-dockerfile-L16). But I don’t know if people build locally and that would cause problems

File: packages/hub/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,13 @@ COPY dist/sql node_modules/graphile-worker/sql
 # This includes the discord bot image assets that are hosted by the hub
 COPY dist/services packages/hub/services
 
+# Copy Prisma necessities
+COPY dist/@prisma/client node_modules/.prisma/client
```
Comment (Aierie) [inline packages/hub/Dockerfile:12]: Is this a typo? If so we might not need the subsequent COPY instructions for `dist/.prisma/<somefile>`?

```suggestion
COPY dist/.prisma/client node_modules/.prisma/client
```

File: packages/hub/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,13 @@ COPY dist/sql node_modules/graphile-worker/sql
...
+COPY dist/@prisma/client node_modules/@prisma/client
+COPY prisma/schema.prisma node_modules/.prisma/client/schema.prisma
+COPY prisma/schema.prisma node_modules/@prisma/client/schema.prisma
+COPY dist/.prisma/client/*.node node_modules/@prisma/client/
```
Comment (Aierie) [inline packages/hub/Dockerfile:16]: Is this a typo? If so we might not need the subsequent COPY instructions for `dist/.prisma/<somefile>`, or for `schema.prisma`? My understanding is `@prisma/client` requires `.prisma/client`, so as long as we copy both of them into `node_modules` we should be ok. This is the code from `@prisma/client/index.js`:

```js
module.exports = {
  ...require('.prisma/client/index'),
}
```

```suggestion
COPY dist/.prisma/client node_modules/.prisma/client
COPY dist/@prisma/client node_modules/@prisma/client
```

Comment (Aierie) [reply packages/hub/Dockerfile:16]: Also just realised we probably don't need to `yarn add @prisma/client` as a final step, as specified in the readme.

**Review by lukemelia — APPROVED** [review]

**Pull request merged**